### DISCUSSION TITLE
Welcome to BRDR and brdrQ Discussions!

### DISCUSSION THREAD

**Discussion by maarten-vermeyen** [OnroerendErfgoed/brdr#32] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community ([BRDR](https://github.com/OnroerendErfgoed/brdr/) and [brdrQ](https://github.com/OnroerendErfgoed/brdrQ/)). We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
     build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by koenedaele** [comment]
Hi! Thank you for this community platform. I'm the IT Director for Flanders Heritage Agency and I've been involved in this project since the start. Mostly here to see if people need help and what the future holds.

**Comment by MichielVansteenkiste** [comment]
Hi everyone,

I'm Michiel Vansteenkiste from the Flemish Environment Agency. We aim to use the BRDR script to accurately define the boundaries of zoning plans. 
I propose **retaining the original attributes** after running the script. Although some data like 'area' and 'length' might no longer be accurate, keeping the original attributes can be useful for reference and validation.

I look forward to your feedback!

**Comment by maarten-vermeyen** [comment]
Hi Michiel,

Welcome to our community and thank you for the suggestion. I've created a separate thread to discuss the feature request in the ideas section: https://github.com/OnroerendErfgoed/brdr/discussions/34#discussion-6945009 .

Kind regards,

Maarten
